source/_patterns/00-atoms/article-title.twig

```diff
@@ -0,0 +1,24 @@
+<div class="article-content__title">
+  <h1>{{ articletitle }}</h1>
```

Comment:
I think this template file should just contain `<h1 class="article-title">{{ articleTitle }}</h1>`. We can always extend it later.

Reply:
Refactored code as discussed.